compiler/src/yul/mappers/expressions.rs

```diff
@@ -38,11 +38,33 @@ pub fn expr(context: &Context, exp: &Spanned<fe::Expr>) -> Result<yul::Expressio
     }
 }
 
-pub fn expr_call(exp: &Spanned<fe::Expr>) -> Result<yul::Expression, CompileError> {
-    if let fe::Expr::Call { args: _, func } = &exp.node {
+pub fn expr_call(
+    context: &Context,
+    exp: &Spanned<fe::Expr>,
+) -> Result<yul::Expression, CompileError> {
+    if let fe::Expr::Call { args, func } = &exp.node {
         if let fe::Expr::Attribute { value: _, attr } = &func.node {
+            let arguments = &args.node;
+            let yul_args = arguments
+                .iter()
+                .map(|val| {
+                    if let fe::CallArg::Arg(arg_expr) = val.node.clone() {
```

Comment:
noting that we have a function that pretty much does the same thing here. it maps a call arg to an expression. I dont think we want to call back into the function mapper, so it might be worth just copying that into here for the time being. later on we'll want to add support for named parameters, so we'll probably create a separate *arguments* module that can deal with this

https://github.com/ethereum/fe/blob/703e31b5c0cf781f9cb10121b2a49d64896cbb19/semantics/src/traversal/functions.rs#L143

Reply:
ah, linked to the semantic analysis version. here's the mapper `call_arg` function

https://github.com/ethereum/fe/blob/703e31b5c0cf781f9cb10121b2a49d64896cbb19/compiler/src/yul/mappers/functions.rs#L138

Reply:
Ah, nice. Maybe we can adopt the snakecharmer Python convention that would go like this:

1. Create a new module `_utils` in `mappers`
2. `_utils` is only meant to be imported from anywhere within `mappers` (which is what the `_` is hinting at)

So, we do not cross import from `expressions` and `functions` but we also don't copy the snippet around.

Reply:
Ah, ok..maybe not a good candidate for `_utils` because the it would need to import from `expressions` which defeats the purpose. But maybe the whole `call_arg` helper should be *moved* into `expressions` because `functions` imports from `expressions` anyway but not the other way around.